### DISCUSSION TITLE
How do I associate .ps1 files with the Powershell executable?

### DISCUSSION THREAD

**Discussion by TechPro424** [PowerShell/PowerShell#19870] [Q&A]
How do I associate .ps1 files with the Powershell executable?
I installed Powershell from the Microsoft Store, and when I double click on .ps1 files, the "Select an app to open this .ps1 file" context menu appears, but I don't see Powershell as an option on it

**Comment by kilasuit** [comment]
Looking into this I have installed PowerShell from the store too (& the preview version) but neither of them can be picked in the picker as part of the Default App settings page either. Will convert this to an issue as so this can be investigated by the team as this is likely something to do with how PowerShell is being installed.
